modules/mergesort/src/mergesort.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>  
+#include "mergesort.h"
+
+using namespace std;
+
+int main(void){
```

Comment:
Testing should be done using Boost test (see the [docs page](http://docs.uwarg.com/computer-vision/Writing-Tests/) for details.